code/modules/client/asset_cache.dm

```diff
@@ -225,7 +225,7 @@ GLOBAL_LIST_EMPTY(asset_datums)
 
 	var/res_name = "spritesheet_[name].css"
 	var/fname = "data/spritesheets/[res_name]"
-	call("rust_g", "file_write")(generate_css(), fname)
+	text2file(generate_css(), fname)

```

Comment:
This was originally here just to be cheeky. It's being removed in order to only use the declared portions of `rust_g`. AFAIK it's never failed which is promising for perf trialling a full replacement sometime in the future.